### DISCUSSION TITLE
Modeling a simple elevator - where can I find a pulley?

### DISCUSSION THREAD

**Discussion by s-celles** [OpenModelica/OpenModelica#11207] [Q&A]
Hello,

I'm looking for help to model an simple elevator

![image](https://github.com/OpenModelica/OpenModelica/assets/109167/321cec14-e85c-4687-96aa-a55ddbc3df18)

The lifting winch and its pulley are driven by a synchronous motor via a speed reducer with ratio red=12.5 (red= Ωm / Ωt) and efficiency η=0.85.

- The diameter of the pulley is D=20cm.
It drives, without slipping, a cable of negligible mass connected on the one hand to the cabin of mass Mcab=100kg and on the other hand a counterweight of mass Mcp=300kg.
- The payload loaded into the cabin is a maximum equal to Mu=400kg.
- The moment of inertia of the winch and the pulley relative to their shaft is Jp=0.1 kg.m².
- The moment of inertia of the motor rotor relative to its axis is Jm=0.05 kg.m².
- The moment of inertia of the mechanical transmitter is neglected.

When moving from one floor to another in the building, the speed changes in accordance with the following profile:

![image](https://github.com/OpenModelica/OpenModelica/assets/109167/4181f3c1-5da5-49f1-8121-8bdc0eb5360f)

I'm looking for a simple model for a pulley ?

Where can I find this kind of model?

Kind regards

**Comment by casella** [comment]
https://www.tandfonline.com/doi/full/10.1080/13873950500071405
